examples/concurrent.ts

```diff
@@ -2,16 +2,16 @@
 // This Node is a socket that replies to hello.js with "world!" when it receives "Hello".
 
 // This example tests concurrency with parallel messages in IPC.
-import { Node } from '../src/index';
+import { Client } from '../src/index';
 import { promisify } from 'util';
 
 const sleep = promisify(setTimeout);
 const TIMES = 10000;
 
-const node = new Node('concurrent')
-	.on('error', (error, client) => console.error(`[IPC] Error from ${client.name}:`, error))
-	.on('socket.disconnect', client => console.error(`[IPC] Disconnected from ${client.name}`))
-	.on('socket.ready', async client => {
+const node = new Client('concurrent')
+	.on('error', (error, client) => console.error(`[IPC] Error from ${client!.name}:`, error))
```

Comment:
Fixed this bug in 877e9e67e81299214807b6fe5062e80f90c4ac51

Can you rebase, please?

Reply:
Same for the `interactive` and `world` examples.

Reply:
Please rename all instances of `client!` to `client`.